services/hidadapter.md

```diff
@@ -0,0 +1,66 @@
+# HID Adapter
+
+    identifier: 0x1e5758b5
+    
+A service for configuring how Jacdac device map to HID input events. Users can have multiple configurations and swap between them using the `swapTo` or `next` commands.
```

Comment:
what commands?

Reply:
removed!

Reply:
I can still see references to `swapTo` which doesn't exist?